README.md

```diff
@@ -57,7 +59,7 @@ Usage
 8. View or optionally download the User security credentials that were created; you will them them later.
 9. Click “Close” to return to the IAM screen.
 10. Click your user name in the list.
-11. Under the Inline Policies header, click the “click here” link to create a new inline policy.
+11. Under the Inline Policies header, click the “AWS IAM role ARN for your AWS Device Farm account. click here” link to create a new inline policy.
```

Comment:
,click the “AWS IAM role ARN for your AWS Device Farm account.”

Reply:
changed